components/mintmaker/base/mintmaker-team.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: mintmaker-maintainers
+  namespace: mintmaker-service
```

Comment:
Is this intended? Or should it be just `mintmaker`? I see in all other places it is `mintmaker` and not `mintmaker-service`.